core/sql/common/ComMisc.cpp

```diff
@@ -171,3 +171,61 @@ NAString ComConvertNativeNameToTrafName (
   return convertedName;
 }
 
+// ----------------------------------------------------------------------------
+// function: ComConvertTrafNameToNativeName
+//
+// this function converts the Trafodion external table name 
+// into its native name format. Both names are in external format.
+//
+// Example:  TRAFODION._HV_HIVE_.abc becomes HIVE.HIVE.abc
+//
```

Comment:
Probably should have double quotes around _HV_HIVE_ since it is in external format.